modules/tasks/tasks/src/main/resources/webapp/css/tasks.css

```diff
@@ -324,6 +324,50 @@ manipulation-sorter .manipulation .remove:before {
     box-shadow: none;
 }
 
+.task-element .arrow-right:before {
+    content: "\F061"
+    font:normal normal normal 14px/1 FontAwesome;
+    /* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#0a4ed6+0,7095e5+42,9fb8ce+46,1957d3+60,4f7cdd+89,89a7ed+100 */
+    background: rgb(10,78,214); /* Old browsers */
+    background: -moz-linear-gradient(top,  rgba(10,78,214,1) 0%, rgba(112,149,229,1) 42%, rgba(159,184,206,1) 46%, rgba(25,87,211,1) 60%, rgba(79,124,221,1) 89%, rgba(137,167,237,1) 100%); /* FF3.6-15 */
+    background: -webkit-linear-gradient(top,  rgba(10,78,214,1) 0%,rgba(112,149,229,1) 42%,rgba(159,184,206,1) 46%,rgba(25,87,211,1) 60%,rgba(79,124,221,1) 89%,rgba(137,167,237,1) 100%); /* Chrome10-25,Safari5.1-6 */
+    background: linear-gradient(to bottom,  rgba(10,78,214,1) 0%,rgba(112,149,229,1) 42%,rgba(159,184,206,1) 46%,rgba(25,87,211,1) 60%,rgba(79,124,221,1) 89%,rgba(137,167,237,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#0a4ed6', endColorstr='#89a7ed',GradientType=0 ); /* IE6-9 */
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
```

Comment:
'Webkit' styles are working only on WebKit browsers. Don't use it when 'standard' style exists.